pom.xml

```diff
@@ -98,13 +100,13 @@
     <dependency>
       <groupId>xml-apis</groupId>
       <artifactId>xml-apis</artifactId>
-      <version>1.4.01</version><!--$NO-MVN-MAN-VER$ -->
```

Comment:
The original version (1.4.01) is actually correct and more recent than 2.0.2.
We need to add a comment to document this confusion.